### ISSUE TITLE
1.8.17: test suite is failing

### ISSUE THREAD

**Issue by kloczek** [doxygen/doxygen#7464]
<pre>+ cd doxygen-1.8.17
+ /usr/bin/make -O -j1 V=1 VERBOSE=1 tests
/usr/bin/cmake -S/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 -B/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/make -f CMakeFiles/Makefile2 tests
make[1]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/cmake -S/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 -B/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/CMakeFiles 84
/usr/bin/make -f CMakeFiles/Makefile2 testing/CMakeFiles/tests.dir/all
make[2]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f src/CMakeFiles/doxycfg.dir/build.make src/CMakeFiles/doxycfg.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src/CMakeFiles/doxycfg.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f src/CMakeFiles/doxycfg.dir/build.make src/CMakeFiles/doxycfg.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;src/CMakeFiles/doxycfg.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[  4%] Built target doxycfg
/usr/bin/make -f libmd5/CMakeFiles/md5.dir/build.make libmd5/CMakeFiles/md5.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libmd5 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libmd5 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libmd5/CMakeFiles/md5.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f libmd5/CMakeFiles/md5.dir/build.make libmd5/CMakeFiles/md5.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;libmd5/CMakeFiles/md5.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[  4%] Built target md5
/usr/bin/make -f liblodepng/CMakeFiles/lodepng.dir/build.make liblodepng/CMakeFiles/lodepng.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/liblodepng /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/liblodepng /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/liblodepng/CMakeFiles/lodepng.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f liblodepng/CMakeFiles/lodepng.dir/build.make liblodepng/CMakeFiles/lodepng.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;liblodepng/CMakeFiles/lodepng.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[  5%] Built target lodepng
/usr/bin/make -f libmscgen/CMakeFiles/mscgen.dir/build.make libmscgen/CMakeFiles/mscgen.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libmscgen /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libmscgen /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libmscgen/CMakeFiles/mscgen.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f libmscgen/CMakeFiles/mscgen.dir/build.make libmscgen/CMakeFiles/mscgen.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;libmscgen/CMakeFiles/mscgen.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 16%] Built target mscgen
/usr/bin/make -f libversion/CMakeFiles/check_git_repository.dir/build.make libversion/CMakeFiles/check_git_repository.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion/CMakeFiles/check_git_repository.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f libversion/CMakeFiles/check_git_repository.dir/build.make libversion/CMakeFiles/check_git_repository.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 17%] <font color="#3465A4"><b>Checking the git repository for changes...</b></font>
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion &amp;&amp; /usr/bin/cmake -D_BUILD_TIME_CHECK_GIT=TRUE -DGIT_WORKING_DIR=/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 -DGIT_EXECUTABLE=/usr/bin/git -DGIT_STATE_FILE=/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/generated_src/git_state -DPRE_CONFIGURE_GIT_VERSION_FILE=/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion/gitversion.cpp.in -DPOST_CONFIGURE_GIT_VERSION_FILE=/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/generated_src/gitversion.cpp -P /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/cmake/git_watcher.cmake
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 17%] Built target check_git_repository
/usr/bin/make -f libversion/CMakeFiles/check_doxygen_version.dir/build.make libversion/CMakeFiles/check_doxygen_version.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion/CMakeFiles/check_doxygen_version.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f libversion/CMakeFiles/check_doxygen_version.dir/build.make libversion/CMakeFiles/check_doxygen_version.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 17%] <font color="#3465A4"><b>Checking the doxygen version for changes...</b></font>
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion &amp;&amp; /usr/bin/cmake -D_BUILD_TIME_CHECK_DOXY=TRUE -DDOXY_STATE_FILE=/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/VERSION -DPRE_CONFIGURE_DOXYGEN_VERSION_FILE=/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion/doxyversion.cpp.in -DPOST_CONFIGURE_DOXYGEN_VERSION_FILE=/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/generated_src/doxyversion.cpp -P /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/cmake/doxygen_version.cmake
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 17%] Built target check_doxygen_version
/usr/bin/make -f libversion/CMakeFiles/doxygen_version.dir/build.make libversion/CMakeFiles/doxygen_version.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/libversion/CMakeFiles/doxygen_version.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f libversion/CMakeFiles/doxygen_version.dir/build.make libversion/CMakeFiles/doxygen_version.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;libversion/CMakeFiles/doxygen_version.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 19%] Built target doxygen_version
/usr/bin/make -f qtools/CMakeFiles/qtools.dir/build.make qtools/CMakeFiles/qtools.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/qtools /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/qtools /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/qtools/CMakeFiles/qtools.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f qtools/CMakeFiles/qtools.dir/build.make qtools/CMakeFiles/qtools.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;qtools/CMakeFiles/qtools.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 32%] Built target qtools
/usr/bin/make -f src/CMakeFiles/generate_configvalues_header.dir/build.make src/CMakeFiles/generate_configvalues_header.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src/CMakeFiles/generate_configvalues_header.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f src/CMakeFiles/generate_configvalues_header.dir/build.make src/CMakeFiles/generate_configvalues_header.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;src/CMakeFiles/generate_configvalues_header.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 32%] Built target generate_configvalues_header
/usr/bin/make -f vhdlparser/CMakeFiles/vhdlparser.dir/build.make vhdlparser/CMakeFiles/vhdlparser.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/vhdlparser /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/vhdlparser /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/vhdlparser/CMakeFiles/vhdlparser.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f vhdlparser/CMakeFiles/vhdlparser.dir/build.make vhdlparser/CMakeFiles/vhdlparser.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;vhdlparser/CMakeFiles/vhdlparser.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 35%] Built target vhdlparser
/usr/bin/make -f src/CMakeFiles/_doxygen.dir/build.make src/CMakeFiles/_doxygen.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src/CMakeFiles/_doxygen.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f src/CMakeFiles/_doxygen.dir/build.make src/CMakeFiles/_doxygen.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;src/CMakeFiles/_doxygen.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 97%] Built target _doxygen
/usr/bin/make -f src/CMakeFiles/doxygen.dir/build.make src/CMakeFiles/doxygen.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/src/CMakeFiles/doxygen.dir/DependInfo.cmake --color=
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f src/CMakeFiles/doxygen.dir/build.make src/CMakeFiles/doxygen.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[3]: Nothing to be done for &apos;src/CMakeFiles/doxygen.dir/build&apos;.
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[ 98%] Built target doxygen
/usr/bin/make -f testing/CMakeFiles/tests.dir/build.make testing/CMakeFiles/tests.dir/depend
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 &amp;&amp; /usr/bin/cmake -E cmake_depends &quot;Unix Makefiles&quot; /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/CMakeFiles/tests.dir/DependInfo.cmake --color=
Dependee &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/CMakeFiles/tests.dir/DependInfo.cmake&quot; is newer than depender &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/CMakeFiles/tests.dir/depend.internal&quot;.
Dependee &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/CMakeFiles/CMakeDirectoryInformation.cmake&quot; is newer than depender &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/CMakeFiles/tests.dir/depend.internal&quot;.
<font color="#75507B"><b>Scanning dependencies of target tests</b></font>
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
/usr/bin/make -f testing/CMakeFiles/tests.dir/build.make testing/CMakeFiles/tests.dir/build
make[3]: Entering directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
[100%] <font color="#3465A4"><b>Running doxygen tests...</b></font>
cd /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing &amp;&amp; /usr/bin/python3 /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/runtests.py --doxygen /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/bin/doxygen --inputdir /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing --outputdir /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing
1..86
Traceback (most recent call last):
  File &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/runtests.py&quot;, line 535, in &lt;module&gt;
    main()
  File &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/runtests.py&quot;, line 532, in main
    sys.exit(testManager.perform_tests())
  File &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/runtests.py&quot;, line 437, in perform_tests
    tester.run(self)
  File &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/runtests.py&quot;, line 402, in run
    self.perform_test(testmgr)
  File &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/runtests.py&quot;, line 237, in perform_test
    data = xpopen(&apos;%s --format --noblanks --nowarning %s&apos; % (self.args.xmllint,check_file))
  File &quot;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/runtests.py&quot;, line 34, in xpopen
    proc = subprocess.Popen(cmd,stdout=subprocess.PIPE,stderr=subprocess.PIPE,encoding=encoding) # Python 3 with encoding
  File &quot;/usr/lib64/python3.8/subprocess.py&quot;, line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File &quot;/usr/lib64/python3.8/subprocess.py&quot;, line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: &apos;xmllint --format --noblanks --nowarning /home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17/testing/test_output_001/out/indexpage.xml&apos;
make[3]: *** [testing/CMakeFiles/tests.dir/build.make:61: testing/CMakeFiles/tests] Error 1
make[3]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[2]: *** [CMakeFiles/Makefile2:833: testing/CMakeFiles/tests.dir/all] Error 2
make[2]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make[1]: *** [CMakeFiles/Makefile2:840: testing/CMakeFiles/tests.dir/rule] Error 2
make[1]: Leaving directory &apos;/home/tkloczko/rpmbuild/BUILD/doxygen-1.8.17&apos;
make: *** [Makefile:408: tests] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.zSAdTV (%check)
</pre>

**Comment by albert-github** [comment]
Do you have `xmllint` installed?

**Comment by kloczek** [comment]
Yes I have
<pre>[tkloczko@barrel ~]$ type xmllint 
xmllint is /usr/bin/xmllint
[tkloczko@barrel ~]$ rpm -qf /usr/bin/xmllint
libxml2-2.9.10-2.fc32.x86_64
</pre>

**Comment by albert-github** [comment]
I tested it with Python 3 as well (currently not at hand, will take a couple of days), I´ll check.

**Comment by albert-github** [comment]
As written I cannot test it at this moment (and also have no *nix system) but in the documentation I saw something that might solve the issue, maybe you can test it (would be much appreciated. It might even be a difference between *nix and Windows!, see https://docs.python.org/3/library/subprocess.html).

Add in the beginning of runtest.py:
 `import shlex`
and replace the line
```
                        proc = subprocess.Popen(cmd,stdout=subprocess.PIPE,stderr=subprocess.PIPE,encoding=encoding) # Python 3 with encoding
```
by
```
                        proc = subprocess.Popen(shlex.split(cmd),stdout=subprocess.PIPE,stderr=subprocess.PIPE,encoding=encoding) # Python 3 with encoding
```

This has probably to be done on multiple places, but first we have to see whether or not this solves the problem.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7470

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.18.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
